tests/measurement/hooks/test_addtask_hook.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Test the AddTask post hook.
+
+"""
+import pytest
+from atom.api import Tuple
+from exopy.tasks.api import RootTask, SimpleTask
+from enaml.workbench.api import Workbench
+from exopy.testing.measurement.dummies import DummyEngine
+from exopy.tasks.tasks.base_views import RootTaskView
+
+from exopy.testing.util import show_widget
+pytest_plugins = str('exopy.testing.tasks.fixtures')
+
+
+@pytest.fixture
+def addtaskhook(measurement):
```

Comment:
You also need to make sure the TaskPlugin is registered. requesting the task_workbench should work.